fusion/java/sample.java

```diff
@@ -0,0 +1,73 @@
+import java.io.IOException;
+import okhttp3.*;
+import okhttp3.Request.Builder;
+import org.json.*;
+
+// using AOuth 2.0 to access Yelp Fusion api
+
+public class sample {
+
+    public static void main(String[] args)throws JSONException, IOException {
+
+
+        // POST https://api.yelp.com/oauth2/token
+        String accessToken=null;
+
+        OkHttpClient client = new OkHttpClient();
+        MediaType mediaType = MediaType.parse("application/x-www-form-urlencoded");
+        RequestBody body = RequestBody.create(mediaType, "client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>&grant_type=client_credentials");
+        Request request = new Request.Builder()
+                .url("https://api.yelp.com/oauth2/token")
+                .post(body)
+                .addHeader("cache-control", "no-cache")
+                .addHeader("postman-token", "8d9de8ad-800c-50e1-fb4a-46fcb5f2f209")
+                .addHeader("content-type", "application/x-www-form-urlencoded")
+                .build();
+
+        try {
+            Response response = client.newCall(request).execute();
+
+            JSONObject jsonObjectToken = new JSONObject(response.body().string().trim());
+
+            accessToken = jsonObjectToken.getString("access_token");
+
+
+        } catch (IOException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+
+        // GET /businesses/search
+        OkHttpClient client2 = new OkHttpClient();
+
+
+        String term = "taco";                       // term
+        String location = "Irivine, CA";            // location
+        String price = "1";                         // price        1 = $, 2 = $$, 3 = $$$, 4 = $$$$
+
+
+        Request request2 = new Builder()
+                .url("https://api.yelp.com/v3/businesses/search?term=" + term + "&location=" + location + "&limit=1&sort_by=rating&price="+price+"")
+                .get()
+                .addHeader("authorization", "Bearer"+" "+accessToken)
```

Comment:
could you replace `accessToken` with `apiKey`, a string that is by default defined as `"<YOUR_API_KEY>"`, to make the query compatible with [our new authentication](https://www.yelp.com/developers/documentation/v3/authentication#where-is-my-client-secret-going)